edu.gemini.lch.model/src/main/java/edu/gemini/lch/model/Interval.scala

```diff
@@ -0,0 +1,28 @@
+package edu.gemini.lch.model
+
+import java.time.Instant
+
+/**
+ * An interval is a period of time between two instants.
+ * Includes start, excludes end.
+ * Requirement: end >= start.
+ * They have fixed ms duration, i.e. end - start and are not comparable.
+ * To compare intervals, you should compare their lengths, i.e. Durations.
+ */
+case class Interval(start: Instant, end: Instant) {
+  def overlaps(other: Interval): Boolean = {
+    val thisStart = start.toEpochMilli
+    val thisEnd   = end.toEpochMilli
+
+    // If null, assume now and check if now falls in the represented interval.
+    if (other == null) {
```

Comment:
This feels a bit iffy. I wonder if we shouldn't rather fail in this case

Reply:
`null` is definitely treated differently in Jave 8 time and joda time. There will invariably be some NPEs that we will come across in this transition and decide hot to handle.